scripts/kfctl.sh

```diff
@@ -35,6 +40,16 @@ createEnv() {
 
 	if [ "${PLATFORM}" == "minikube" ]; then
 	  echo KUBEFLOW_CLOUD=minikube >> ${ENV_FILE}
+    set +x
+    infer_minikube_settings
+    set -x
+    download_kubeflow_source
```

Comment:
Why are you calling download_kubeflow_source here?
The needed source will already be there; source is downloaded by
https://github.com/kubeflow/kubeflow/blob/master/scripts/download.sh

The source is located based on the location of kfctl.sh

Once you get rid of it it will use your local code and you can test it.

Reply:
For the local experience, I don't like the idea of a user downloading the kubeflow source first. In fact, in its current form, they only need to download kfctl.sh, and then:
kfctl.sh init appName --platform minikube
cd appName
../kfctl.sh generate all
../kfctl.sh apply all